repairnator/repairnator-core/src/main/java/fr/inria/spirals/repairnator/config/RepairnatorConfig.java

```diff
@@ -74,6 +74,9 @@
     private Duration summaryFrequency;
     private String[] notifySummary;
     private int numberOfPatchedBuilds;
+    private Boolean kubernetesMode;
```

Comment:
primitive `boolean` would be enough instead of an object `Boolean` (upper b)

Reply:
Hi :), it is fixed .